client/src/modals/GetMemberEmailsModal.js

```diff
@@ -8,7 +8,7 @@ import Box from '@mui/material/Box';
 function GetMemberEmailsModal(props) {
   return (
     <Modal open={props.getEmailsModal} >
-        <Box className='ModalBackground'>
+        <Box className='ModalBackground' style={{width: "370px"}}>
```

Comment:
Maybe change the width to a responsive value, by using vw units instead. Not a real problem since we're not really making the application responsive anyway, but just wanted to mention it :D